pytd/tests/test_writer.py

```diff
@@ -59,9 +69,9 @@ def test_cast_dtypes_keep_list(self):
         self.assertEqual(
             dtypes, set([np.dtype("int"), np.dtype("float"), np.dtype("O")])
         )
-        self.assertTrue(self.dft["H"].apply(lambda x: isinstance(x, list)).all())
-        self.assertTrue(self.dft["I"].apply(lambda x: isinstance(x, list)).all())
-        self.assertTrue(self.dft["J"].apply(lambda x: isinstance(x, list)).all())
+        self.assertTrue(self.dft["H"].apply(_check_column_type, args=[list]).all())
+        self.assertTrue(self.dft["I"].apply(_check_column_type, args=[list]).all())
+        self.assertTrue(self.dft["J"].apply(_check_column_type, args=[list]).all())
```

Comment:
It works, but, since [the official documentation requires `args` in the `tuple` type](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.apply.html), let's make our code more accurate:

```py
_check_column_type, args=(list,)
```

Reply:
fixed d97e189